README.md

```diff
@@ -152,10 +152,18 @@ metadata:
   name: fenceagentsremediationtemplate-default
   namespace: default
 spec:
-  template: {}
+  template:
+    spec:
+        remediationStrategy: <remediation_strategy>
 ```
 
 > *Note*: FenceAgentsRemediationTemplate CR must be created in the same namespace that the FAR operator has been installed.
+ 
+The `.spec.template.spec.remediation_strategy` field can either be `ResourceDeletion` or `OutOfServiceTaint`:
```

Comment:
It can be the `.spec.template.spec.remediation_strategy` field of fartemplate CR or simply `.spec.remediation_strategy` of FAR CR. I prefer the latter and modify the README in the [FAR CR fields](https://github.com/medik8s/fence-agents-remediation/pull/145/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R197-R202) rather than the template spec of fartemplate